### PR TITLE
add required registry

### PR DESCRIPTION
**Pull request by cosmic-byte**
## Description
**Problem** - import fails for missing argument in `_list_packages` private function.

Screenshot 2020-02-19 at 12 42 07 PM 

**Solution** - Add default registry to `_list_packages` usage [here](https://github.com/quiltdata/quilt/blob/a764778d029713e235296b3c7d1e62f3678afcfd/api/python/quilt3/imports.py#L46)

### CODE REVIEW

**Comment by mistercrunch** [comment]
# [Codecov](https://codecov.io/gh/quiltdata/quilt/pull/1537?src=pr&el=h1) Report
> Merging [#1537](https://codecov.io/gh/quiltdata/quilt/pull/1537?src=pr&el=desc) into [master](https://codecov.io/gh/quiltdata/quilt/commit/0a4a3b1f979b5d5ec4b587fa980b5d0c9e77886d&el=desc) will **decrease** coverage by `0.01%`.
> The diff coverage is `n/a`.

[Impacted file tree graph](https://codecov.io/gh/quiltdata/quilt/pull/1537?src=pr&el=tree)

```diff
@@            Coverage Diff             @@
##           master    #1537      +/-   ##
==========================================
- Coverage   83.33%   83.31%   -0.02%     
==========================================
  Files          27       27              
  Lines        3715     3717       +2     
==========================================
+ Hits         3096     3097       +1     
- Misses        619      620       +1     
```


| [Impacted Files](https://codecov.io/gh/quiltdata/quilt/pull/1537?src=pr&el=tree) | Coverage Δ | |
|---|---|---|
| [quilt3/search\_util.py](https://codecov.io/gh/quiltdata/quilt/pull/1537/diff?src=pr&el=tree#diff-cXVpbHQzL3NlYXJjaF91dGlsLnB5) | `75.86% <0.00%> (-3.45%)` | :arrow_down: |
| [quilt3/packages.py](https://codecov.io/gh/quiltdata/quilt/pull/1537/diff?src=pr&el=tree#diff-cXVpbHQzL3BhY2thZ2VzLnB5) | `89.20% <0.00%> (+0.01%)` | :arrow_up: |
| [quilt3/imports.py](https://codecov.io/gh/quiltdata/quilt/pull/1537/diff?src=pr&el=tree#diff-cXVpbHQzL2ltcG9ydHMucHk=) | `97.05% <0.00%> (+0.08%)` | :arrow_up: |

------

[Continue to review full report at Codecov](https://codecov.io/gh/quiltdata/quilt/pull/1537?src=pr&el=continue).
> **Legend** - [Click here to learn more](https://docs.codecov.io/docs/codecov-delta)
> `Δ = absolute <relative> (impact)`, `ø = not affected`, `? = missing data`
> Powered by [Codecov](https://codecov.io/gh/quiltdata/quilt/pull/1537?src=pr&el=footer). Last update [0a4a3b1...0550098](https://codecov.io/gh/quiltdata/quilt/pull/1537?src=pr&el=lastupdated). Read the [comment docs](https://docs.codecov.io/docs/pull-request-comments).

**Comment by akarve** [comment]
@cosmic-byte there are some open todos, is this unintentional?

**Comment by cosmic-byte** [comment]
> @cosmic-byte there are some open todos, is this unintentional?

They were unintentional. Removed.

File: api/python/quilt3/packages.py [hunk 1]
```diff
@@ -521,7 +521,7 @@ def browse(cls, name, registry=None, top_hash=None):
         return cls._browse(name=name, registry=registry, top_hash=top_hash)
 
     @classmethod
-    def _browse(cls, name, registry, top_hash):
+    def _browse(cls, name, registry=None, top_hash=None):
```
Comment (sir-sigurd) [inline api/python/quilt3/packages.py]: Is this change needed for the fix?

File: api/python/quilt3/imports.py [hunk 2]
```diff
@@ -41,9 +41,10 @@ def exec_module(cls, module):
 
         elif len(name_parts) == 3:  # e.g. module.__name__ == quilt3.data.foo
             namespace = name_parts[2]
+            registry_passed = PhysicalKey.from_url(registry)
```
Comment (sir-sigurd) [inline api/python/quilt3/imports.py]: It looks like it's usually called `registry_parsed` in the codebase:
```suggestion
            registry_parsed = PhysicalKey.from_url(registry)
```

File: api/python/quilt3/packages.py (hunk 1 above)
Comment (cosmic-byte) [reply api/python/quilt3/packages.py]: Not in this context, but would also break execution [here](https://github.com/quiltdata/quilt/blob/c0dd45ac6d47d21345991690ee042f3d00c25787/api/python/quilt3/imports.py#L49) without a default argument. Similar to the issue with the required argument.

File: api/python/quilt3/imports.py (hunk 2 above)
Comment (cosmic-byte) [reply api/python/quilt3/imports.py]: You are right.
